topics/create-first-app.md

```diff
@@ -19,8 +19,8 @@ Here you will learn how to create and run your first KMM application.
 
     ![Mobile Multiplatform project - general settings](kmm-project-wizard-2.png)
 
-5. Keep the default names for the application and shared folders, select the checkbox to generate sample tests for your project, 
-and click **Finish**.  
+5. Keep the default names for the application and shared folders. Select the checkbox to generate sample tests for your 
```

Comment:
Too many actions as separate sentences (Keep + Select + Select) for a single item. If it's important to explain in a single list item, we could use the inner list:

Smth like:

In the window that opens, do the following:
* Keep the default names ...
* Select this ...
* Select that ...

Click Finish to create a new project ...

Use a list with numbers if it's important to emphasize the order of actions.